iOS/Flows/Conversation/Conversation Header/MembersViewController.swift

```diff
@@ -39,28 +42,37 @@ class MembersViewController: DiffableCollectionViewController<MembersCollectionV
         ConversationsManager.shared.$activeConversation
             .removeDuplicates()
             .mainSink { [unowned self] conversation in
-                Task {
-                    guard let cid = conversation?.cid else {
-                        await self.dataSource.deleteAllItems()
+                self.loadDataTask?.cancel()
+
+                if let cid = conversation?.cid {
+                    self.conversationController = ConversationController.controller(cid)
+                } else {
+                    self.conversationController = nil
+                }
+
+                self.loadDataTask = Task { [weak self] in
```

Comment:
Weak self! It's actually really great for one-off Tasks like this.

Long story short, if you want a Task to stop working immediately after we deinit, then use weak self.